cmd/erasure-server-pool-decom.go

```diff
@@ -556,8 +556,15 @@ func (z *erasureServerPools) Init(ctx context.Context) error {
 }
 
 func (z *erasureServerPools) decommissionObject(ctx context.Context, bucket string, gr *GetObjectReader) (err error) {
-	defer gr.Close()
 	objInfo := gr.ObjInfo
+
+	defer func() {
+		gr.Close()
+		if err == nil {
```

Comment:
we should also log failed operations @vadmeste

Reply:
> we should also log failed operations @vadmeste

Yes you are right, I added a new field called 'Error' in the audit logging for that